### ISSUE TITLE
youtube-dl -U ERROR: can't find the current version. Please try again later.

### ISSUE THREAD

**Issue by clpacheco90** [ytdl-org/youtube-dl#21375]
## Checklist

- [X ] I'm asking a question
- [X ] I've looked through the README and FAQ for similar questions
- [X ] I've searched the bugtracker for similar questions including closed ones

## Question

When I try to use ```youtube-dl.exe -U``` in my internal company network it's giving me an error ```ERROR: can't find the current version. Please try again later.```. But in the same machine if I change the current network to an external network (like my cellphone network 4G) I can update the youtube-dl.
Do you know which URLs are be using by youtube-dl? In my internal network, all youtube URLs are able to access.

**Comment by dstftw** [comment]
https://yt-dl.org/update/LATEST_VERSION

**Comment by LiquidLightning** [comment]
I have had an updating problem ever since a few versions ago, and the issue is present using the current 2019.07.16 version.

I am using the build available for OS X El Capitan 10.11.6.

```
$ youtube-dl --version
2019.07.16
$ sudo youtube-dl -U -v
[debug] System config: []
[debug] User config: [u'--ffmpeg-location', u'/usr/local/bin/ffmpeg', u'--audio-quality', u'0', u'-i', u'-w', u'--merge-output-format', u'mkv', u'--no-part', u'--username', u'PRIVATE', u'--password', u'PRIVATE']
[debug] Custom config: []
[debug] Command-line args: [u'-U', u'-v']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2019.07.16
[debug] Python version 2.7.10 (CPython) - Darwin-15.6.0-x86_64-i386-64bit
[debug] exe versions: ffmpeg N-93875-gc2771bbf81-tessus
[debug] Proxy map: {}
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/update.py", line 45, in update_self
    newversion = opener.open(VERSION_URL).read().decode('utf-8').strip()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 431, in open
    response = self._open(req, data)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 449, in _open
    '_open', req)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 409, in _call_chain
    result = func(*args)
  File "/usr/local/bin/youtube-dl/youtube_dl/utils.py", line 2724, in https_open
    req, **kwargs)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 1197, in do_open
    raise URLError(err)
URLError: <urlopen error EOF occurred in violation of protocol (_ssl.c:590)>

ERROR: can't find the current version. Please try again later.
$```
